lib/v2/screens/explore_screens/plant_seeds/interactor/viewmodels/plant_seeds_bloc.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:bloc/bloc.dart';
+import 'package:seeds/v2/blocs/rates/viewmodels/rates_state.dart';
+import 'package:seeds/v2/datasource/local/settings_storage.dart';
+import 'package:seeds/v2/domain-shared/page_state.dart';
+import 'package:seeds/v2/screens/explore_screens/plant_seeds/interactor/mappers/seeds_amount_change_mapper.dart';
+import 'package:seeds/v2/screens/explore_screens/plant_seeds/interactor/mappers/user_balance_state_mapper.dart';
+import 'package:seeds/v2/screens/explore_screens/plant_seeds/interactor/viewmodels/bloc.dart';
+import 'package:seeds/v2/screens/transfer/send_enter_data/interactor/usecases/get_available_balance_use_case.dart';
+
+/// --- BLOC
+class PlantSeedsBloc extends Bloc<PlantSeedsEvent, PlantSeedsState> {
+  PlantSeedsBloc(RatesState rates) : super(PlantSeedsState.initial(rates));
+
+  @override
+  Stream<PlantSeedsState> mapEventToState(PlantSeedsEvent event) async* {
+    if (event is LoadUserBalance) {
+      yield state.copyWith(pageState: PageState.loading);
+      Result result = await GetAvailableBalanceUseCase().run(settingsStorage.accountName);
```

Comment:
Idk if we really need here a call I mean we already have this data from ExploreScreen, this screen fetch the following data:
```
    var futures = [
      _balanceRepository.getBalance(account),
      _plantedRepository.getPlanted(account),
      _voiceRepository.getAllianceVoice(account),
      _voiceRepository.getCampaignVoice(account)
    ];
```
I think I just can pass by navigation this data.

Reply:
In fact how often do we need refresh the Explore screen data, in fact actually each one of the three main screes fires its initial load data when the bottombar is changed.

Well.. do we need the fresh data when user navigates to plant seeds?

Reply:
humm. i rather make this call. Get the latest data.